src/components/panes/CanvassAssignmentPane.scss

```diff
@@ -1,7 +1,10 @@
 .CanvassAssignmentPane.PaneBase {
     min-width: 450px;
-}
 
+    @include small-screen {
+        min-width: 100%;
+    }
+}
```

Comment:
Could be replaced by solution proposed for CallPane

Reply:
Fixed!